.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 dist
 
 src/.DS_Store
-.DS_Store
\ No newline at end of file
+.DS_Store
+collector-portal-framework-0.0.0-development.tgz
```

Comment:
tgz filen behövs ju egentligen inte alls tänker jag?

Reply:
Exactly 👍 that's why we should ignore it so that we accidentally don't commit it.